test/e2e/framework.go

```diff
@@ -972,6 +972,24 @@ func (data *TestData) createNginxPodOnNode(name string, nodeName string) error {
 	}, false, nil)
 }
 
+// createAgnhostPodOnNodeWithMultiPort creates a Pod in the test namespace with
+// a single agnhost container listening multiple ports.
+// The Pod will be scheduled on the specified Node (if nodeName is not empty).
+func (data *TestData) createAgnhostPodOnNodeWithMultiPort(name string, nodeName string, ports []int32) error {
```

Comment:
this is a very specific function, I think it could be kept in test/e2e/networkpolicy_test.go

Reply:
Done.